src/pages/library/DataBrowser.js

```diff
@@ -33,117 +63,204 @@ const sidebarSections = [{
+        columns: [
+          {
+            header: div({ className: 'sr-only' }, ['Select dataset']),
+            size: { basis: 37, grow: 0 }, key: 'checkbox'
+          }, {
+            header: div({ style: styles.table.header }, [h(MiniSortable, { sort, field: 'Dataset Name', onSort: setSort }, ['Dataset Name'])]),
+            size: { grow: 2.2 }, key: 'name'
+          }, {
+            header: div({ style: styles.table.header }, [h(MiniSortable, { sort, field: 'Project', onSort: setSort }, ['Project'])]),
+            size: { grow: 1 }, key: 'project'
+          }, {
+            header: div({ style: styles.table.header }, [h(MiniSortable, { sort, field: 'No. of Subjects', onSort: setSort }, ['No. of Subjects'])]),
+            size: { grow: 1 }, key: 'subjects'
+          }, {
+            header: div({ style: styles.table.header }, [h(MiniSortable, { sort, field: 'Data Type', onSort: setSort }, ['Data Type'])]),
+            size: { grow: 1 }, key: 'dataType'
+          }, {
+            header: div({ style: styles.table.header }, [h(MiniSortable, { sort, field: 'Last Updated', onSort: setSort }, ['Last Updated'])]),
+            size: { grow: 1 }, key: 'lastUpdated'
+          }
+        ],
+        rowStyle: styles.table.row,
+        cellStyle: { border: 'none', paddingRight: 15 },
+        useHover: false,
+        underRowKey: 'underRow',
+        rows: _.map(datum => {
+          const { project, dataType, locked } = datum
+          return {
+            checkbox: h(Checkbox, {
+              'aria-label': datum['dct:title'],
+              checked: _.includes(datum, selectedData),
+              onChange: () => toggleSelectedData(datum)
+            }),
+            name: h(Link,
+              { onClick: () => Nav.goToPath('library-details', { id: datum['dct:identifier'] }) },
+              [datum['dct:title']]
+            ),
+            project,
+            subjects: datum?.counts?.donors,
+            dataType,
+            lastUpdated: Utils.makeStandardDate(datum['dct:modified']),
+            underRow: div({ style: { display: 'flex', alignItems: 'flex-start', paddingTop: '1rem' } }, [
+              div({ style: { display: 'flex', alignItems: 'center' } }, [
+                locked ?
+                  h(ButtonSecondary, {
+                    style: { height: 'unset', textTransform: 'none', fontWeight: 'bold' },
+                    onClick: () => setRequestDatasetAccessList([datum])
+                  }, [icon('lock'), div({ style: { paddingLeft: 10, paddingTop: 5, fontSize: 12 } }, ['Request Access'])]) :
+                  div({ style: { color: colors.success(), display: 'flex', fontWeight: 'bold' } }, [icon('unlock'), div({ style: { paddingLeft: 10, paddingTop: 5, fontSize: 12 } }, ['Open Access'])])
+              ])
+            ])
+          }
+        }, sortData(listData))
+      })])
+  }
+
+  return h(FooterWrapper, { alwaysShow: true }, [
+    libraryTopMatter('browse & explore'),
```

Comment:
If these items are components, lets render them using `h` so react knows they are components and can optimize:
You can do this in a follow up PR to keep the size down
```js
h(LibraryTopMatter, ['browse & explore`])
```